src/client/pydaos/raw/daos_api.py

```diff
@@ -1468,13 +1466,14 @@ def create(self, poh, con_uuid=None, con_prop=None, cb_func=None):
         if self.cont_input_values.type != "Unknown":
             self.cont_prop.dpp_entries[idx].dpe_type = ctypes.c_uint32(
                 DaosContPropEnum.DAOS_PROP_CO_LAYOUT_TYPE.value)
-            if self.cont_input_values.type == "posix":
+            if self.cont_input_values.type in ("posix", "POSIX"):
```

Comment:
Optional: we could also do self.cont_input_values.type.lower() == "posix"

Reply:
I did use .casefold() initially, but that's a python3 function, .lower() would probably work on python2 and 3, I've still got a test window open here where I tried that.  This should apply to the hdf5 line as well, and I'm not keen on failing through to a default if there's a string provided but it's unknown either, which is why I've added a TODO on L1476

Reply:
Ok